sspl_test/alerts/self_hw/self_hw_utilities.py

```diff
@@ -63,3 +64,32 @@ def get_manufacturer_name():
         if search_res:
             manufacturer = search_res.groups()[0]
     return manufacturer
+
+def get_server_details():
+    """Returns dictionary of node server information which includes
+    FRU device descriptions like bmc manufacturer, product and more.
+    """
+    fru_info = {
+        "Host": socket.getfqdn(),
+        "Board Mfg": None,
+        "Board Product": None,
+        "Board Part Number": None,
+        "Product Name": None,
+        "Product Part Number": None
+        }
+    cmd = "ipmitool fru print"
+    search_res = ""
+    res_op, _, res_rc = SimpleProcess(cmd).run()
+    if isinstance(res_op, bytes):
+        res_op = res_op.decode("utf-8")
+    if res_rc == 0:
+        search_res = re.search(r"((.*[\S\n\s]+ID 1\)).*)|(.*[\S\n\s]+)", res_op)
```

Comment:
We need to extract from "FRU Device (ID 0)", here seems you are look for ID1

Please add code comment here that code looking for 
FRU Device Description : Builtin FRU Device (ID 0)

Reply:
Regex here is to grep information between starting string to "ID 1".
So ideally ID 0 will be grepped.
Sure I will add the comment.

Reply:
Done.

Reply:
Ok